src/guides/v2.3/frontend-dev-guide/templates/template-email-migration.md

```diff
@@ -226,6 +226,6 @@ it will resolve to “John”.
 
 <!-- Link Definitions -->
 [Insert Variable]: https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/templates/template-email.html#customize-content
-[New Order email template]: https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Sales/view/frontend/email/order_new.html
-[model]: https://github.com/magento/magento2ce/blob/2.3-develop/app/code/Magento/Sales/Model/Order/Email/Sender/InvoiceSender.php
-[1]: https://github.com/magento/magento2ce/blob/2.3-develop/app/code/Magento/Email/Model/AbstractTemplate.php
+[New Order email template]: https://github.com/magento/magento2/blob/2.3/app/code/Magento/Sales/view/frontend/email/order_new.html
+[model]: https://github.com/magento/magento2ce/blob/2.3/app/code/Magento/Sales/Model/Order/Email/Sender/InvoiceSender.php
```

Comment:
`https://github.com/magento/magento2ce` repository in not available

Reply:
It's a private repo.

Reply:
Ha...false positive because I am logged in.